log/logger.go

```diff
@@ -112,11 +117,16 @@ func (l *Logger) Enabled() bool {
 // log will log the message. It DOES NOT check Enabled() first so that should
 // be checked by it's callers
 func (l *Logger) log(msg string, context map[string]interface{}) {
+	_, file, line, _ := runtime.Caller(3)
+	file = filepath.Base(filepath.Dir(file)) + "/" + filepath.Base(file)
```

Comment:
Suggest computing the path like this:
```
func sourcePath(file string) string {
	if wd, err := os.Getwd(); err == nil {
		if rel, err := filepath.Rel(wd, file); err == nil && !strings.HasPrefix(rel, "../") {
			return rel
		}
	}
	return filepath.Join(filepath.Base(filepath.Dir(file)), filepath.Base(file))
}

```
That way, if your working directory is inside the source tree, you'll get a valid relative path that an editor can follow. If you're outside, fall back to your format.